cmd/log_test.go

```diff
@@ -0,0 +1,155 @@
+// Copyright 2022 The Witness Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"os"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+	"github.com/sirupsen/logrus/hooks/test"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_newLogger(t *testing.T) {
+	logger := newLogger()
+	require.NotNil(t, logger)
+	require.NotNil(t, logger.l)
+
+	// Check if the logger has the expected configuration
+	assert.Equal(t, os.Stderr, logger.l.Out)
+
+	// Validate formatter settings
+	formatter, ok := logger.l.Formatter.(*logrus.TextFormatter)
+	require.True(t, ok)
+	assert.True(t, formatter.DisableLevelTruncation)
+	assert.True(t, formatter.PadLevelText)
+	assert.True(t, formatter.DisableTimestamp)
+}
+
+func Test_logrusLogger_SetLevel(t *testing.T) {
+	logger := newLogger()
+
+	// Test valid log levels
+	validLevels := []string{
+		"debug", "info", "warn", "error", "fatal", "panic",
+	}
+
+	for _, level := range validLevels {
+		err := logger.SetLevel(level)
+		require.NoError(t, err)
+	}
+
+	// Test invalid log level
+	err := logger.SetLevel("invalid-level")
+	require.Error(t, err)
+	assert.Contains(t, err.Error(), "not a valid logrus Level")
+}
+
+func Test_logrusLogger_LoggingMethods(t *testing.T) {
```

Comment:
If we want to test the logging methods, we should test the functions from `log.go` instead of implementing the functions here.